### PR TITLE
Workaround issue where some homebrew (like moppi-flower) would erroneously load into kernel ram where it won't fit

### PR DESCRIPTION
**Pull request by hrydgard**
Not sure if this is the best way, but works fine... @unknownbrackets ?

### CODE REVIEW

**Comment by unknownbrackets** [comment]
Hmm, this makes sense.  I suppose a kernel module is probably allowed to load anywhere it wants.... haven't tested this.

-[Unknown]

**Comment by hrydgard** [comment]
Yeah. I think this should be safe and fixes some homebrew, so I'm merging.

File: Core/ELF/ElfReader.cpp [hunk 1]
```diff
@@ -413,6 +413,11 @@ int ElfReader::LoadInto(u32 loadAddress, bool fromTop)
 		}
 	}
 	totalSize = totalEnd - totalStart;
+
+	// If a load address is specified that's in regular RAM, override kernel module status
+	bool inUser = totalStart > 0x08900000;
```
Comment (unknownbrackets) [inline Core/ELF/ElfReader.cpp]: `>= PSP_GetUserMemoryBase()` might be preferrable I guess, not sure what 0x08900000 indicates.

-[Unknown]

Comment (hrydgard) [inline Core/ELF/ElfReader.cpp]: Right, yeah, that would be more elegant.

**Pull request merged**